From 465fdb18516c464a62c1063d3576f9a6b7cd6550 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Thu, 2 Feb 2017 21:35:26 +0900
Subject: [PATCH 1700/2066] Revert "watchdog: renesas-wdt: Change DDR backup
 function names"

This reverts commit ef17b72474da0fd456685325ccc4237c47fa4733 ("watchdog:
renesas-wdt: Change DDR backup function names").

Since a timer count is re-setup with a timeout value by resume, backup
of a register is unnecessary.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 drivers/watchdog/renesas_wdt.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/watchdog/renesas_wdt.c b/drivers/watchdog/renesas_wdt.c
index 3c7708d..d14ac2c 100644
--- a/drivers/watchdog/renesas_wdt.c
+++ b/drivers/watchdog/renesas_wdt.c
@@ -232,7 +232,7 @@ static int rwdt_suspend(struct device *dev)
 	if (!rwdt_ip.virt_addr)
 		rwdt_ip.virt_addr = priv->base;
 
-	ret = rcar_handle_registers(&rwdt_ip, DO_BACKUP);
+	ret = handle_registers(&rwdt_ip, DO_BACKUP);
 #endif /* CONFIG_RCAR_DDR_BACKUP */
 	return ret;
 }
@@ -241,7 +241,7 @@ static int rwdt_resume(struct device *dev)
 {
 	int ret = 0;
 #ifdef CONFIG_RCAR_DDR_BACKUP
-	ret = rcar_handle_registers(&rwdt_ip, DO_RESTORE);
+	ret = handle_registers(&rwdt_ip, DO_RESTORE);
 #endif /* CONFIG_RCAR_DDR_BACKUP */
 	return ret;
 }
-- 
1.7.5.4

